corehq/apps/case_importer/views.py

```diff
@@ -265,6 +334,7 @@ def excel_fields(request, domain):
         'case_field_specs': case_field_specs,
         'domain': domain,
         'mirroring_enabled': mirroring_enabled,
+        'is_bulk_import': request.POST.get('is_bulk_import', 'False') == 'True',
```

Comment:
Just checking: Should these boolean values be strings?

Reply:
Since this is from the frontend, it's coming in as a string so we need to compare the boolean value of the string. We can convert them to booleans but I thought it would be simpler to just compare the strings themselves as this is being done in a few other places as well.